script.js

```diff
@@ -349,6 +349,11 @@ let projectData = [
     projectImage: "assets/GIFs/wordScramble.gif",
     projectUrl: "public/wordScramble.html",
   },
+  {
+    projectName: "Puzzle quiz",
+    projectImage: "assets/Images/quiz",
```

Comment:
Add a proper image. It;s not showing
![image](https://user-images.githubusercontent.com/38184186/115065056-4c63d600-9f0b-11eb-9997-46c4e45ee9e7.png)